src/scripts/store/dimStoreItem.directive.template.html

```diff
@@ -30,6 +30,7 @@
          width="44" />
   </div>
   <div ng-class="::vm.badgeClassNames"
+       class="{{ vm.item.dtrRating || vm.item.quality ? 'item-stat-no-bg':'' }}"
```

Comment:
Unfortunately I don't think you can have multiple ng-class directives. We might have to make the whole `badgeClassNames` no longer bind-once and modify it.